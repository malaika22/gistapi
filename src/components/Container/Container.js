import styled from 'styled-components';
import PropTypes from 'prop-types';

export const Container = ({ children }) => {
  return <ContainerWrapper>{children}</ContainerWrapper>;
};

const ContainerWrapper = styled.div`
  max-width: 672px;
  margin: auto;
`;

Container.propTypes = {
  children: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.node), PropTypes.node]).isRequired
};
