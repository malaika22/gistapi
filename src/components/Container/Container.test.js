import { render } from '@testing-library/react';
import { Container } from './Container';

describe('Container', () => {
  it('should render successfully', () => {
    const { baseElement } = render(<Container />);
    expect(baseElement).toBeTruthy();
  });
  it('should match snapshot', () => {
    const { asFragment } = render(<Container />);
    expect(asFragment()).toMatchSnapshot();
  });
});
