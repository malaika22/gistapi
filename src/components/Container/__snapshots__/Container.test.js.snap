// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-bgqQcB cTRgPa"
  />
</DocumentFragment>
`;

exports[`Container should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-bgqQcB cTRgPa"
  />
</DocumentFragment>
`;
